docs/release-notes/observability.st

```diff
@@ -0,0 +1,9 @@
+:orphan:
+
+**Improvements**
+
+-  Kubernetes: Added Determined resource information - e.g., workspace, task ID, etc - as pod labels.
+
+-  Configuration: Added DCGM helm chart, prometheus configuration, and two dashboards: API Monitoring and Resource Utilization to tools/observability directory.
+
+-  Documentation: Added Kubernetes specific guide for enabling observability.
```

Comment:
:orphan:

**Improvements**

- Kubernetes: Add Determined resource information such as ``workspace`` and ``task ID`` as pod labels. This improvement facilitates better resource tracking and management within Kubernetes environments.

- Configuration: Introduce a DCGM Helm chart and Prometheus configuration to the ``tools/observability`` directory. Additionally, two new dashboards, "API Monitoring" and "Resource Utilization", have been added to improve observability and operational insight.

Reply:
(++ avoid documenting the documentation)